.changeset/hungry-clouds-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
```

Comment:
Should we go with a Minor bump as this is a new feature?

Reply:
Yeah we probably do need minor, good catch @anagstef !